src/WindowsAuth/README.md

```diff
@@ -0,0 +1,70 @@
+Integrated Windows Authentication Sample App
+====
+
+.NET api app for testing Integrated Windows Auth on CF
+
+
+Deploy
+=======
+
+Run the following command to deploy:
+
+```sh
+cf push windows-auth -s windows -b hwc_buildpack
+```
+
+Requirements
+=======
+This sample app requires at least 512mb of memory to run on CloudFoundry.
+
+
+Endpoints
+=======
+1. `GET /` Howdy!
+1. `GET /auth` Returns the name and authentication type of the authenticated user
+
+
+Authenticating
+======
+To authenticate via the browser, simply visit the `/auth` endpoint and you'll be prompted for credentials.  For example, if your domain is `MSFT` and your credentials are `bgates:m0n3ybag$`, you would enter `MSFT\bgates` as your username and `m0n3ybag$` as your password.
+
+To use `curl`, you would do something like
+```
+curl --ntlm --user 'MSFT\bgates:m0n3ybag$' https://auth.apps.<cf-instance>/auth
+```
+
+
+Building
+=============
+
+#### Requirements:
+
+* Microsoft Windows OS
+
+* [Msbuild.exe](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild)
+
+#### Build
+
+##### Method 1
+
+* Make sure you have `msbuild.exe` on your `$PATH`. (If you're using a VM created using a [BOSH Stemcell for Windows](https://bosh.io/stemcells), it will available at `$env:WINDIR\Microsoft.NET\Framework64\v*\MSBuild.exe`)
+
+* Make your code changes
+
+* Run `./make.bat`
+
+* This is will build the app and you'll see it has a new `bin/` directory.
+
+##### Method 2
+
+You can also use the dotnet-framework docker image which has `msbuild` preinstalled to easily build nora.
```

Comment:
What is nora?

Reply:
Oops!  The [other windows app in CATs](https://github.com/cloudfoundry/cf-acceptance-tests/tree/develop/assets/nora) whose readme I stole :)

Reply:
Removed final reference to nora